honeybadger.go

```diff
@@ -24,8 +49,53 @@ func (hb *HoneyBadger) sendBatch() {
 }
 
 // Create batch polling random transaction in queue
-func (hb *HoneyBadger) createBatch() {
-	panic("implement me w/ test case :-)")
+func (hb *HoneyBadger) createBatch() ([]Transaction, error) {
+	buf, err := loadBuf(hb, min(hb.b, hb.que.Len()))
+	if err != nil {
+		return nil, err
+	}
+	pol, pushQ := selectRand(hb, buf, int(hb.b/hb.n))
+	defer pushQ()
+
+	return pol, nil
+}
+
+func loadBuf(hb *HoneyBadger, minVal int) ([]Transaction, error) {
```

Comment:
plz rename this function!

and `minVal` has no meaning in this function context! plz rename `minVal` too